cmd/daytona/config/ssh_file.go

```diff
@@ -118,74 +122,157 @@ func generateSshConfigEntry(profileId, workspaceId, projectName, knownHostsPath
 		tab+"StrictHostKeyChecking no\n"+
 		tab+"UserKnownHostsFile %s\n"+
 		tab+"ProxyCommand \"%s\" ssh-proxy %s %s %s\n"+
-		tab+"ForwardAgent yes\n\n", projectHostname, knownHostsPath, daytonaPath, profileId, workspaceId, projectName)
+		tab+"ForwardAgent yes\n", projectHostname, knownHostsPath, daytonaPath, profileId, workspaceId, projectName)
 
-	return config, nil
-}
+	if gpgForward {
+		localSocket, err := getLocalGPGSocket()
+		if err != nil {
+			log.Warn(err)
```

Comment:
Let's do `log.Trace` here (and below on remote sock) actually.

If we use warn, it always gets logged to the user on workspace create but it's expected to fail since the workspace hasn't started yet.

![Screenshot 2024-10-16 at 13 01 04](https://github.com/user-attachments/assets/ce5fc3dc-8617-4c3a-8756-048d6755c87b)

Reply:
done